<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvDialog</class>
 <widget class="QWidget" name="ConvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KML/GPX Converter</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="Panel1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="TimeSpan">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Time Span</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QComboBox" name="TrackColor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDateEdit" name="TimeY2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLabel" name="Label5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Q-Flags</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="TimeInt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Waypoint</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Track</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QComboBox" name="PointColor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>By Q-Flag</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="Label7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Input/Output/GE Exe File</string>
        </property>
       </widget>
      </item>
      <item row="9" column="8">
       <widget class="QPushButton" name="BtnGoogleEarthFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="TimeIntF">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Interval (s)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="2">
       <widget class="QLineEdit" name="Offset2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7" colspan="2">
       <widget class="QLineEdit" name="Offset3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QCheckBox" name="Compress">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>.kmz</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="9">
       <widget class="QLabel" name="Message">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="9">
       <widget class="QLineEdit" name="OutputFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="Offset1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" colspan="3">
       <widget class="QComboBox" name="QFlags">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q=6</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="6" colspan="3">
       <widget class="QComboBox" name="OutputAlt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ellipsoidal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Geodetic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QTimeEdit" name="TimeH1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="7" colspan="2">
       <widget class="QTimeEdit" name="TimeH2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDateEdit" name="TimeY1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>2</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QLabel" name="Label4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Time</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="9">
       <widget class="QLineEdit" name="InputFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QLabel" name="Label3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Altitude</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="8">
       <widget class="QLineEdit" name="GoogleEarthFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="6" colspan="3">
       <widget class="QComboBox" name="OutputTime">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GPST</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UTC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JST</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="8">
       <widget class="QPushButton" name="BtnInputFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="AddOffset">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Add Offset E/N/U (m)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output Format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QRadioButton" name="FormatKML">
        <property name="text">
         <string>Google Earth &amp;KML</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QRadioButton" name="FormatGPX">
        <property name="text">
         <string>GP&amp;X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnView">
        <property name="text">
         <string>&amp;View....</string>
        </property>
        <property name="icon">
         <iconset resource="rtkpost_qt.qrc">
          <normaloff>:/buttons/doc.bmp</normaloff>:/buttons/doc.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnConvert">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Con&amp;vert</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnClose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rtkpost_qt.qrc"/>
 </resources>
 <connections/>
</ui>
